com.projectrk.launcher.yml

```diff
@@ -0,0 +1,232 @@
+
+  - --persist=.
+  - --filesystem=xdg-download
+  # should fix access to SD card on the deck
+  - --filesystem=/run/media
+  # should fix steamdeck controler navigation
+  - --filesystem=/run/udev:ro
+
+  - --allow=devel
+  - --allow=multiarch
+  - --allow=per-app-dev-shm
+
+  - --device=all
+
+  - --env=DBUS_FATAL_WARNINGS=0
+  - --env=PATH=/app/bin:/app/utils/bin:/usr/bin
+
+  # --- Steam ---
+  # Pressure Vessel
+  # See https://github.com/flathub/com.valvesoftware.Steam/commit/0538256facdb0837c33232bc65a9195a8a5bc750
+  - --env=XDG_DATA_DIRS=/app/share:/usr/lib/extensions/vulkan/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/share:/usr/lib/pressure-vessel/overrides/share
+  - --env=XDG_CONFIG_DIRS=/etc/xdg:/usr/lib/x86_64-linux-gnu/GL:/usr/lib/i386-linux-gnu/GL
+  - --env=PROTON_DEBUG_DIR=/var/tmp
+
+  # UMU
+  - --env=UMU_LOG=debug
+  - --env=GAMEID=0
+  - --env=PROTONPATH=GE-Proton
+
+  - --env=WINEPREFIX=/var/data/project-rk
+  - --env=WINEDLLOVERRIDES='DDraw.dll,D3DImm.dll=n,b'
+
+  - --env=TZ=
+  - --unset-env=TZ
+  - --env=SDL_VIDEODRIVER=
+  - --unset-env=SDL_VIDEODRIVER
+
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: *runtime-version
+
+  org.freedesktop.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    version: *runtime-version
+    no-autodownload: true
+
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: *gl-version
+    versions: *gl-versions
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: *gl-merge-dirs
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+    autoprune-unless: active-gl-driver
+
+  org.freedesktop.Platform.GL32.Debug:
+    directory: lib/debug/lib/i386-linux-gnu/GL
+    version: *gl-version
+    versions: *gl-versions
+    subdirectories: true
+    no-autodownload: true
+    merge-dirs: *gl-merge-dirs
+    enable-if: active-gl-driver
+    autoprune-unless: active-gl-driver
+
+  org.freedesktop.Platform.VAAPI.Intel.i386:
+    directory: lib/i386-linux-gnu/dri/intel-vaapi-driver
+    version: *runtime-version
+    versions: *runtime-version
+    autodelete: false
+    no-autodownload: true
+    add-ld-path: lib
+    download-if: have-intel-gpu
+    autoprune-unless: have-intel-gpu
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: *runtime-version
+    no-autodownload: true
+    autodelete: false
+
+  org.freedesktop.Platform.ffmpeg_full.i386:
+    directory: lib32/ffmpeg
+    add-ld-path: .
+    version: *runtime-version
+    no-autodownload: true
+    autodelete: false
+
+  com.valvesoftware.Steam.CompatibilityTool:
+    subdirectories: true
+    directory: share/steam/compatibilitytools.d
+    version: stable
+    versions: stable;beta;test
+    no-autodownload: true
+    autodelete: true
+
+  com.valvesoftware.Steam.Utility:
```

Comment:
why are these needed?

Reply:
This is leveraging umu, which is an alternative wrapper to Valve's Steam Linux Runtime (pressure vessel).

These Steam components assist the pressure vessel Vulkan-Loader.